### ISSUE TITLE
transaction sign multisig-transfer default signature directory is not working

### ISSUE THREAD

**Issue by jondubois** [Capitalisk/ldpos-commander#40]
Location directory of the signatures do not use ~ (Default: /home/jon/.local/share/ldpos-commander/signatures/)

- This is supposed to say `Location directory of the signatures to use`
- If I press enter without typing anything, it says `File written in undefined/signature_315eb49a6db46f485c6de66604847966a15098c5_clskbde48efe4fb34f91f3bfad4e4e8feb081b302b0f.json`
 - Right now it uses `undefined/` as the directory name if user presses enter (instead of using the default).

Make sure that it creates the directory `/home/jon/.local/share/ldpos-commander/signatures/` recursively if it doesn't already exist (similar to `mkdir -p`).

**Comment by maarteNNNN** [comment]
Changed prompt to:
```
CAUTION: Tilde (~) does not work!
Location directory of the signatures to use (Default: ...)
```
in 30b44b03fdf06ba19ef5ff80fe93c28cf5f25e62

**Comment by maarteNNNN** [comment]
@jondubois try running `ldpos config clean signatures default-path`. This will remove the directory's config file.
I added the fix in 192ff057f822dffda9ec655afdd44aa3dad0d906 and 11815c3ac351469759c5e2be7088270010c16725
